src/Avalonia.Themes.Default/Avalonia.Themes.Default.csproj

```diff
@@ -7,6 +7,7 @@
     <ProjectReference Include="..\Avalonia.Animation\Avalonia.Animation.csproj" />
     <ProjectReference Include="..\Avalonia.Base\Avalonia.Base.csproj" />
     <ProjectReference Include="..\Avalonia.Controls\Avalonia.Controls.csproj" />
+    <ProjectReference Include="..\Avalonia.Controls.DataGrid\Avalonia.Controls.DataGrid.csproj" />
```

Comment:
Adding this reference here will cause all projects to reference `Avalonia.Controls.DataGrid`, which is what putting it in its own assembly was intended to avoid...

It would probably be better to extract the default theme from this assembly and put it in the `Avalonia.Controls.DataGrid` assembly.